src/config/settings/base.py

```diff
@@ -197,6 +197,7 @@
         # https://docs.djangoproject.com/en/dev/ref/settings/#dirs
         "DIRS": [
             str(BASE_DIR / "templates"),
+            str(BASE_DIR / "templates/forum"),
```

Comment:
This line seems less than ideal to me, since we are already including the templates directory. However, I see why you did it this way. Because of where django-machina is expecting the templates to be, putting them straight into the templates directory could get messy, and this prevents that by keeping them contained in their own directory. I wish we could find a cleaner solution though.

Reply:
To override pre made django-machina html templates we have to add files to the projects templates folder that have the same name and we have to follow the same file structure as the django-machina templates folder. The issue is that this means for the program to actually override the file in question, board_base.html, without telling the program to specifically search in the templates/forum folder we would have to store board_base.html in the base of the main projects templates folder. This is messy and creates organization issues. The solution to this is to put it in a folder called forums and tell the program to search that folder for it.

Reply:
All of that being said if we are fine keeping this line:  str(BASE_DIR / "templates/forum"), in the settings file then I will definitely add a comment to it.

Reply:
Comment looks good, you might also add something to the effect of "... discover the correct templates without cluttering the templates base directory" to clarify further the necessity of this line.